src/main/java/edu/tamu/modspineomatic/config/model/Credentials.java

```diff
@@ -0,0 +1,28 @@
+package edu.tamu.modspineomatic.config.model;
+
+public class Credentials {
```

Comment:
If we are using Lombok, lets add annotation/s to not have to code the constructor and getter and setters.